src/components/AddMap.js

```diff
@@ -75,7 +27,12 @@ const LeafletMap = ({ location, setLocation, center }) => {
           className="map-tiles"
         />
         <UserMarker />
-        <PositionMarker />
+        <PositionMarker
+          markerLocation={markerLocation}
+          setMarkerLocation={setMarkerLocation}
```

Comment:
Im not a big fan of prop-drilling both value and setter.
Maybe useContext?

Reply:
Got it!